README.md

```diff
@@ -41,10 +41,9 @@ If you use Hail for published work, please cite both the software:
 
 and the forthcoming manuscript describing Hail:
 
- - Cotton Seed, Alex Bloemendal, Jonathan M Bloom, Jacqueline I Goldstein, Timothy Poterba.  Hail: An Open-Source Framework for Scalable Genetic Data Analysis.  In preparation.
+ - Cotton Seed, Alex Bloemendal, Jonathan M Bloom, Jacqueline I Goldstein, Daniel King, Timothy Poterba.  Hail: An Open-Source Framework for Scalable Genetic Data Analysis.  In preparation.
 
-of the following paper which includes a brief introduction to Hail in
```

Comment:
of should have been or.  I want people to reference the in preparation paper and, if not allowed, Andrea's paper.